source/demo/on-content-script-invalidated.ts

```diff
@@ -1,4 +1,4 @@
-import {onContextInvalidated} from 'webext-events';
+import {onContextInvalidated} from '../index.js';
```

Comment:
The TypeScript resolver was rewritten in Parcel 2.9 and this is a regression 🤷‍♂️